app/github_repo_fetcher.rb

```diff
@@ -47,6 +48,14 @@ def all_alphagov_repos
     end
   end
 
+  def latest_commit(app_name, path)
```

Comment:
Sorry for the late comment, which I missed initially: is there a plan to reuse this for the README as well? While I agree it's a separate change to how we review docs, since it currently has no concept of review, it does make sense to expose the same kind of information for it - although maybe as a separate page.

Did you have any thoughts on doing this for READMEs?

Reply:
It would make sense to do it for READMEs too, though I think that's out of scope for this PR. The implementation might end up being different too, if the `.readme` method returns the commit sha, for example.